content/cpp/concepts/constructors/constructors.md

```diff
@@ -16,75 +16,121 @@ CatalogContent:
 
 A **constructor** is a specially defined method in a C++ class that is automatically called when an instance of that class is created. It is typically used for tasks such as initializing class attributes in a new object. Like with functions, a constructor can take arguments that can aid in the initialization. The arguments are passed to the constructor method when a new object is created.
 
-### Syntax and Examples
+## Syntax
 
-A constructor method is defined within a class by using the `ClassName` followed by parentheses.
+A constructor method is defined within a class by using the `ClassName` followed by parentheses. It does not have a return type.
+
+```cpp
+class  ClassName {
+  public:
+    ClassName() {
+      // Constructor code
+    }
+};
+```
+
+## Examples
+
+- Basic Constructor
+
+A basic constructor, also known as a default constructor, is a special type of constructor in C++ that takes no parameters. The compiler will automatically provide a default constructor if there is no constructor explicitly defined for a class. This constructor is used for initializing an object when no specific values are needed, and it also initializes member variables to default values.
 
 ```cpp
 class MyClass {
   public:
-    // Define a constructor that outputs a message
-    MyClass() {
-      std::cout << "The constructor was executed!";
+    // Define a basic constructor
+    MyClass () {
+      std::cout << "The constructor was executed!" << std::endl;
```

Comment:
In C++, there should not be a space between the class name and the parentheses when defining a constructor or invoking a function.

Also, the new line is not required at the end, the example is already complete.

```suggestion
    // Define a basic constructor
    MyClass() {
      std::cout << "The constructor was executed!";
```